equipment-service/src/main/java/com/telecominfraproject/wlan/equipment/controller/EquipmentController.java

```diff
@@ -416,20 +416,6 @@ public Equipment updateCellSizeAttributes(@RequestBody EquipmentCellSizeAttribut
                             cellSizeAttributes.getMulticastRate()));
                     radioConfig.setManagementRate(SourceSelectionManagement.createAutomaticInstance(
                             cellSizeAttributes.getManagementRate()));
-                } else {
```

Comment:
Perhaps we should throw an exception here? Otherwise the caller will think the request was applied.